### DISCUSSION TITLE
What does `active` means in the json configs

### DISCUSSION THREAD

**Discussion by melmasri** [felixleopoldo/benchpress#32] [General]
I've seen in many places the command `active` , here is an example, 

```{
  "id": "omcmc_itsample-bge",
  "burn_in": 0,
  "thinning": 1,
  "functional": [
    "score",
    "size"
  ],
  "active": true
}
```
What does it mean? do you mean executing the command, or running it? if so, why not use `execute` instead?

**Comment by felixleopoldo** [comment]
Hi, its because the mcmc objects have quite many fields so removing and writing and writing again can be quite time consuming. So I made an optional flag just for disabling.

**Reply by melmasri** [reply]
ok maybe we should use something like `activate_module`, a bit more expressive.

**Comment by felixleopoldo** [comment]
Sure, or maybe `deactivated` as this flag is optional.
